lstchain/io/io.py

```diff
@@ -1345,3 +1347,9 @@ def get_mc_fov_offset(filename):
 
     return mean_offset
 
+
+def get_resource_path(filename):
+    """
+    Get a resource data path in lstchain package
+    """
+    return Path(resource_filename("lstchain", filename))
```

Comment:
Use importlib.resources:

```
return files("lstchain") / filename
```

Reply:
https://docs.python.org/3/library/importlib.resources.html

Reply:
Any particular reason?
`resource_filename` was used everywhere in the code, I just created a generic function instead of copy/pasting the same code again.

Reply:
H just say @morcuended comment, thanks